definitions/features/service.rb

```diff
@@ -99,9 +99,15 @@ def allowed_action?(action)
     %w[start stop restart status enable disable].include?(action)
   end
 
-  def filter_services(service_list, options)
-    service_list = include_unregistered_services(service_list, options[:include])
+  def extend_service_list_with_sockets(service_list, options)
+    return service_list unless options[:include_sockets]
+
+    socket_list = service_list.map(&:socket).compact.select(&:exist?)
+    service_list.concat(socket_list)
+    service_list
```

Comment:
Would it be cleaner not to modify `service_list` in place?
```suggestion
    service_list + socket_list
```